common/frame.go

```diff
@@ -548,7 +551,17 @@ func (f *Frame) waitForFunction(
 	return result, nil
 }
 
-func (f *Frame) waitForSelector(selector string, opts *FrameWaitForSelectorOptions) (*ElementHandle, error) {
+func (f *Frame) waitForSelector(selector string, opts *FrameWaitForSelectorOptions) (h *ElementHandle, err error) {
```

Comment:
Better to name this `waitForSelectorWithRetry(selector string, opts *FrameWaitForSelectorOptions, retry int)` and leave the original `waitForSelector`'s name as it was.

Reply:
Yeah, makes sense. Would you want to expose the `retry` argument to JS instead of having it constant?